Description: Make error message bad_git_dependency test less strict
Author: Vasudev Kamath <vasudev@copyninja.info>
Bug: https://github.com/rust-lang/cargo/issues/4908
Last-Update: 2018-01-24

--- a/tests/bad-config.rs
+++ b/tests/bad-config.rs
@@ -389,7 +389,7 @@
   failed to clone into: [..]
 
 Caused by:
-  [[..]] 'file:///' is not a valid local file URI
+  [..]'file:///' is not a valid local file URI[..]
 "));
 }
 
